### PR TITLE
[Meta] Test new warning for duplicate autoload keys in rootmaps

### PR DESCRIPTION
**Pull request by dpiparo**
This is the test for issue #13431.

### CODE REVIEW

**Comment by dpiparo** [comment]
See ROOT PR https://github.com/root-project/root/pull/13478

File: root/meta/rootmap/duplicateWarning/README.txt [hunk 1]
```diff
@@ -0,0 +1,5 @@
+17-08-2023
+
+This test has been put in one directory by itself since it deals with rootmap files.
+The warning messages that the interpreter can prompt while reading rootmap files is tested. Unfortunately, if any other test is put in this directory, all rootmaps are read and a warning is prompted which
```
Comment (pcanal) [inline root/meta/rootmap/duplicateWarning/README.txt:4]:
```suggestion
The warning messages that the interpreter can prompt while reading rootmap files is tested. Unfortunately, if any other test is put in this directory, all rootmaps are read and a warning is printed which 
```

Comment (dpiparo) [reply root/meta/rootmap/duplicateWarning/README.txt:4]: thx. Will be there as soon as all other little quirks are fixed.

**Review by pcanal — APPROVED** [review]
LGTM.  Note that the branch name here `dpiparo:issue_13432` is distinct from the one `dpiparo:issue_13431` in the related PR (https://github.com/root-project/root/pull/13478) (off by one)

**Comment by dpiparo** [comment]
yes, it's a typo :(. Let me try something. Apologies for this mistake.

**Comment by dpiparo** [comment]
Superseeded by https://github.com/root-project/roottest/pull/1003, with the correct branch name in order for [this PR](https://github.com/root-project/root/pull/13478) to ROOT to pick it up for testing.

**Pull request closed without merging**